airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml

```diff
@@ -27,6 +27,22 @@ data:
     oss:
       enabled: true
   releaseStage: generally_available
+  releases:
+    breakingChanges:
+      2.0.0:
+        message: "All Ads-Insights-* streams now have updated schemas."
```

Comment:
can you add the user action needed here?

Reply:
updated